rules/import-index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const isImportingIndex = m => m.endsWith('/') || m.endsWith('index.js') || m.endsWith('index');
```

Comment:
Related to this, I'd like to see a few valid tests with all kinds of imported modules, to make sure nothing breaks

'foobar', 'index', 'fooindex', 'indexbar', (add an at sign before the next 2) 'index/foo', 'foo/index.js', 'index.js', './foobar/' and more if you can think of some.